src/components/Wizard/CreateVmWizard/NetworksTab.js

```diff
@@ -420,3 +427,22 @@ NetworksTab.propTypes = {
   namespace: PropTypes.string.isRequired,
   isCreateRemoveDisabled: PropTypes.bool,
 };
+
+// TODO refactor to use immutables and partial state changes + validation
+const stateToProps = (state, { networkConfigs, wizardReduxId }) => ({
+  namespace: getVmSettingValue(state, wizardReduxId, NAMESPACE_KEY),
+  sourceType: getVmSettingValue(state, wizardReduxId, PROVISION_SOURCE_TYPE_KEY),
+  isCreateRemoveDisabled: isVmwareProvider(state, wizardReduxId),
+  networks: immutableListToShallowJS(getNetworks(state, wizardReduxId)),
+  networkConfigs: immutableListToShallowJS(networkConfigs),
+});
+
+const dispatchToProps = (dispatch, props) => ({
+  onChange: (networks, valid, locked) =>
+    dispatch(vmWizardActions[types.setNetworks](props.wizardReduxId, networks, valid, locked)),
+});
+
+export const ConnectedNetworksTab = connect(
```

Comment:
Network and Storage Tab were not changed - just connected to redux. We can make other optimizations in the future (like moving the validation to reducer)